static/app/components/devtoolbar/components/alerts/alertsPanel.tsx

```diff
@@ -0,0 +1,185 @@
+import AlertRuleStatus from 'sentry/views/alerts/list/rules/alertRuleStatus';
+import type {Incident, MetricAlert} from 'sentry/views/alerts/types';
+import {CombinedAlertType} from 'sentry/views/alerts/types';
+import {alertDetailsLink} from 'sentry/views/alerts/utils';
+
+import useConfiguration from '../../hooks/useConfiguration';
+import {
+  badgeWithLabelCss,
+  gridFlexEndCss,
+  listItemGridCss,
+  listItemPlaceholderWrapperCss,
+} from '../../styles/listItem';
+import {panelInsetContentCss, panelSectionCss} from '../../styles/panel';
+import {resetFlexColumnCss, resetFlexRowCss} from '../../styles/reset';
+import {smallCss, xSmallCss} from '../../styles/typography';
+import InfiniteListItems from '../infiniteListItems';
+import InfiniteListState from '../infiniteListState';
+import PanelLayout from '../panelLayout';
+import SentryAppLink from '../sentryAppLink';
+import useTeams from '../teams/useTeams';
+
+import useInfiniteAlertsList from './useInfiniteAlertsList';
+
+export default function AlertsPanel() {
+  const {projectId, projectSlug, trackAnalytics} = useConfiguration();
+  const queryResult = useInfiniteAlertsList();
+
+  const estimateSize = 108;
+  const placeholderHeight = `${estimateSize - 8}px`; // The real height of the items, minus the padding-block value
+
+  return (
+    <PanelLayout title="Alerts">
+      <div css={[smallCss, panelSectionCss, panelInsetContentCss]}>
+        <span css={[resetFlexRowCss, {gap: 'var(--space50)'}]}>
+          Active Alerts in{' '}
+          <SentryAppLink
+            to={{url: `/projects/${projectSlug}/`}}
+            onClick={() => {
+              trackAnalytics?.({
+                eventKey: `devtoolbar.alerts-list.header.click`,
+                eventName: `devtoolbar: Click alert-list header`,
+              });
+            }}
+          >
+            <div css={[resetFlexRowCss, {display: 'inline-flex', gap: 'var(--space50)'}]}>
+              <ProjectBadge
+                css={css({'&& img': {boxShadow: 'none'}})}
+                project={{slug: projectSlug, id: projectId}}
+                avatarSize={16}
+                hideName
+                avatarProps={{hasTooltip: false}}
+              />
+              {projectSlug}
+            </div>
+          </SentryAppLink>
+        </span>
+      </div>
+
+      <div css={resetFlexColumnCss}>
+        <InfiniteListState
+          queryResult={queryResult}
+          backgroundUpdatingMessage={() => null}
+          loadingMessage={() => (
+            <div
+              css={[
+                resetFlexColumnCss,
+                panelSectionCss,
+                panelInsetContentCss,
+                listItemPlaceholderWrapperCss,
+              ]}
+            >
+              <Placeholder height={placeholderHeight} />
+              <Placeholder height={placeholderHeight} />
+              <Placeholder height={placeholderHeight} />
+              <Placeholder height={placeholderHeight} />
```

Comment:
can probably reuse this too!

Reply:
i'll come back and see about some of the copy+paste in a followup. but playing with this `css={}` prop is interesting so far.